src/components/Accordion/Accordion.css.js

```diff
@@ -144,7 +155,25 @@ export const TitleUI = styled('div')`
     .${titleClassNames.iconCaretClassName} {
       color: ${getColor('text.slightlyMuted')};
     }
+
+    .Accordion__Heading {
+      color: #253642;
+    }
+
+    .Accordion__Subheading {
+      color: #556575;
```

Comment:
colorway is telling me that `charcoal.400` would be the closest color for that one. Would that work @digitaldesigner ?